nemo/utils/exp_manager.py

```diff
@@ -798,7 +798,7 @@ def configure_checkpointing(
     if params.dirpath is None:
         params.dirpath = Path(log_dir / 'checkpoints')
     if params.filename is None:
-        params.filename = f'{name}--{{{params.monitor}:.2f}}-{{epoch}}'
+        params.filename = f'{name}--{{{params.monitor}:.4f}}-{{epoch}}'
```

Comment:
This is fine with me. @titu1994 , any thoughts?

Reply:
This is good. Multiplying by 100 is problematic cause you have to remember the scale up in downstream stuff.